spec/JSON-RPC.md

```diff
@@ -1417,19 +1419,20 @@ No parameters
         }
       ],
       "v":0
-    }
-  ],
+    }]
+  },
   "id":null
 }
 ```
 
 [Back to **List of methods**](#list-of-methods)
 
 ## chain_getPendingTransactionsCount
-Returns a count of the transactions that are in the current transaction queue.
+Returns a count of the transactions whose insertion_timestamps are in the given range from the transaction queues.
```

Comment:
Returns a count of the transactions whose insertion_timestamps are in the given range from the transaction queues -> Returns a count of the transactions that have insertion_timestamps within the given range from the transaction queues.

Reply:
Fixed. Thank you.